### DISCUSSION TITLE
[Done] Let us add public pgp key for encrypt outgoing mail on reversed alias.

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#687] [Feature Requests]
I use my personal domain email on your services, and use many reversed alias like that i can send mail directly form protonmail and or ctemplar directly to people i know.

But some of them have pgp enabled service (by their own, or protonmail / ctemplar etc), it would be nice to have a way to upload their public key of their reversed alias to encrypt the sended message from your server.

Note here is different from : https://github.com/simple-login/app/discussions/429

This request is to add the key of the person whom i send a mail to.

**Comment by nguyenkims** [comment]
@BirdInFire hey you can add a PGP key for a contact and SimpleLogin will encrypt the email you send to them. To add a PGP key for contact, you can click on the "Edit" button on this contact:

![contact-pgp](https://user-images.githubusercontent.com/3167780/142396433-1eea497b-c057-44c2-9586-191abbd2da0f.png)

**Reply by ghost** [reply]
how i haven't see it sorry
